modules/core/src/main/scala/org/tessellation/infrastructure/trust/storage/TrustStorage.scala

```diff
@@ -205,23 +201,22 @@ object TrustStorage {
           } else store
         }
 
-      private def getBiasedTrust(trust: TrustMap): TrustMap =
-        seedlistScores.fold(trust) { e =>
-          val peerIds = e.keys
-
-          val updatedTrust = peerIds
-            .filterNot(trust.trust.contains)
-            .map(_ -> TrustInfo(predictedTrust = config.seedlistInputBias.some))
-            .toMap
-          val updatedPeerLabels = peerIds
-            .filterNot(trust.peerLabels.value.contains)
-            .map(_ -> PublicTrust.empty)
-            .toMap
-
-          TrustMap(
-            trust.trust ++ updatedTrust,
-            PublicTrustMap(trust.peerLabels.value ++ updatedPeerLabels)
-          )
-        }
+      private def getBiasedTrust(trust: TrustMap): TrustMap = {
+        val peerIds = seedlistScores.keys
+
+        val updatedTrust = peerIds
+          .filterNot(trust.trust.contains)
+          .map(_ -> TrustInfo(predictedTrust = config.seedlistInputBias.some))
+          .toMap
```

Comment:
Not necessary. `Map.++` only requires an `Iterable[_]` which is what you have before the `.toMap`

Reply:
Removed.